pkg/transaction/definitions.go

```diff
@@ -50,20 +50,22 @@ const (
 
 // LogEvent - Log event to be sent to Condor
 type LogEvent struct {
-	Version            string   `json:"version"`
-	Stamp              int64    `json:"timestamp"`
-	TransactionID      string   `json:"transactionId"`
-	Environment        string   `json:"environment,omitempty"`
-	APICDeployment     string   `json:"apicDeployment,omitempty"`
-	EnvironmentName    string   `json:"environmentName,omitempty"`
-	EnvironmentID      string   `json:"environmentId"`
-	TenantID           string   `json:"tenantId"`
-	TrcbltPartitionID  string   `json:"trcbltPartitionId"`
-	Type               string   `json:"type"`
-	TargetPath         string   `json:"targetPath,omitempty"`
-	ResourcePath       string   `json:"resourcePath,omitempty"`
-	TransactionEvent   *Event   `json:"transactionEvent,omitempty"`
-	TransactionSummary *Summary `json:"transactionSummary,omitempty"`
+	Version                string   `json:"version"`
+	Stamp                  int64    `json:"timestamp"`
+	TransactionID          string   `json:"transactionId"`
+	Environment            string   `json:"environment,omitempty"`
+	APICDeployment         string   `json:"apicDeployment,omitempty"`
+	EnvironmentName        string   `json:"environmentName,omitempty"`
+	EnvironmentID          string   `json:"environmentId"`
+	TenantID               string   `json:"tenantId"`
+	TrcbltPartitionID      string   `json:"trcbltPartitionId"`
+	Type                   string   `json:"type"`
+	TargetPath             string   `json:"targetPath,omitempty"`
+	ResourcePath           string   `json:"resourcePath,omitempty"`
+	TransactionEvent       *Event   `json:"transactionEvent,omitempty"`
+	TransactionSummary     *Summary `json:"transactionSummary,omitempty"`
+	UnRedactedTargetPath   string
```

Comment:
Should add json tag((`json:"-"`) to not serialize these properties

Reply:
Fixed it in another commit.